apps/thunderhub/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  thub:
+    image: apotdevin/thunderhub:v0.12.0
```

Comment:
This should be pinned to a specific sha-256

Reply:
While the image should be pinned to the SHA-256 digest of it's multi-arch manifest list, we don't expect the app developers to do that. We'll pin it ourselves as a part of the final changes before merge:

> After you've submitted your app, we'll review your pull request, create the required Tor hidden services for it, make some adjustments in the docker-compose.yml file, such as removing any port conflicts with other apps, pinning Docker images to their sha256 digests, assigning unique IP addresses to the containers, etc before merging. 
>
> https://github.com/getumbrel/umbrel/blob/master/apps/README.md